EK-427 Status wysyłki do Tety

vitaliicherednichenko opened the pull request:
[EK-427](https://yesbizuteria.atlassian.net/browse/EK-427)

### Description

Zintegruję gem spree_comments z aplikacją verona-spree.

Dodam miejsce w panelu administracyjnym w zamówieniach, gdzie będzie można przeglądać historię komentarzy.

Dodam miejsce w panelu admina, gdzie będzie można ustawić typ wiadomości Spree::CommentType

Naprawię błąd związany z tworzeniem nowego typu wiadomości

## Type of change

- [ ] Bug fix 🐛
- [ ] New feature ✨
- [ ] Breaking change 🚨
- [ ] Documentation update 📚
- [ ] Refactor 🔨

### How to test

- FIXME: Add the steps describing how to test your changes manually
- ...

### Screenshots

<img width="1172" height="665" alt="Screenshot 2025-07-29 at 11 09 56" src="https://github.com/user-attachments/assets/67de3b2a-3328-4824-8145-43031760b73a" />


### Video



https://github.com/user-attachments/assets/a2c09b61-5155-4dff-9b05-50e191d06e10




## Checklist ✅

- [ ] I have tested my code locally.
- [ ] I have updated documentation where necessary.
- [ ] I have linked the related issue (if any).

### Review

- [ ] Verify all copy with a content writer
- [ ] Get approval from at lest 2 team members (including Patryk).


### Other

Provide additional notes, remarks, links, mention specific people to review if need.

Commit:
EK-427 Status wysylki do Tety

Commit:
EK-427 Add orders index messages

vitaliicherednichenko commented:
@pkatek proszę utworzyć branch `develop`

tamaraStrzeszewska23 commented:
1. Update readme
2. Na statusach odpowiedzi od tety są puste pola, nie powinno być coś typu 'czekujące'?
3. Co z kwestią ręcznego wpisywania komentarzy do zamówienia? Bo kojarzę jak Patryk pokazywał to w poniedziałek na przykładzie shopify, a nie pamiętam konkluzji czy tutaj też powinno to być

Commit:
EK-427 Update REDME, add badges and translations

vitaliicherednichenko commented:
> 1. Update readme
> 2. Na statusach odpowiedzi od tety są puste pola, nie powinno być coś typu 'czekujące'?
> 3. Co z kwestią ręcznego wpisywania komentarzy do zamówienia? Bo kojarzę jak Patryk pokazywał to w poniedziałek na przykładzie shopify, a nie pamiętam konkluzji czy tutaj też powinno to być

1. Updated
2. Updated
3. Nie ma możliwości ręcznego utworzenia wiadomości do zamówienia, wszystko dzieje się automatycznie w SendOrderJob i AnswersWorker

tamaraStrzeszewska23 commented on config/locales/pl.yml:
```diff
@@ -15,5 +15,7 @@ pl:
-    teta_answer: Odpowiedź Tety
-    sent: Wysłano
+    teta:
+      answer: Odpowiedź TETA
```
Nie chcę wyjść na czepialskiego xD ale jeśli to potwierdzenie otrzymane od TETY, to może poprawniej by było 'Potwierdzone' albo 'Otrzymane'? Tak, żeby to był przymiotnik zamiast rzeczownika

vitaliicherednichenko replied:
Doceniam Twoją opinię i uważam, że Twoja wersja jest lepsza, ale nie chcę otrzymywać kolejnego tłumaczenia od biznesa po kolejnej korekcie

tamaraStrzeszewska23 commented on README.md:
```diff
@@ -19,7 +19,7 @@ Installation
 
     gem 'spree_comments', github: 'cloudsailor/spree_comments'
-    gem 'acts_as_commentable', github: 'fatfreecrm/acts_as_commentable', branch: 'rails-61'
+    gem 'acts_as_commentable', git: 'https://github.com/jibranusman95/acts_as_commentable.git'
```
tutaj też chyba trzeba będzie dać referencję do naszego sforkowanego repo

agnieszkajacek commented on Gemfile:
```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-gem 'spree', '>= 4.4'
+gem 'spree', '~> 5.0', '>= 5.0.5'
```
To już raczej nie zadziała, na `verona-spree` mamy już najnowszą wersję spree https://github.com/cloudsailor/verona-spree/blob/develop/Gemfile#L121

Sugeruję tutaj również podnieść wersję do najnowszej w ramach spójności

Commit:
EK-427 Split migrations, upgrade gems

tamaraStrzeszewska23 reviewed (commented):
Mamy nowe migracje, a nie widać zmian w schema.db, nie zostało puszczone rails db:mgirate?

agnieszkajacek commented on README.md:
```diff
@@ -1,7 +1,7 @@
 ==============
 
-* It only supports Spree 4.4, Rails 6.1, Ruby 3.0.2, not tested on other versions.
+* It only supports Spree 5.0.3, Rails 7.2, Ruby 3.3.8, not tested on other versions.
```
Dlaczego mowa tutaj o Spree 5.0.3, jeśli w Gemfile zostało dodane wyższe Spree?

nika-piotrowska replied:
Podbijam temat

vitaliicherednichenko replied:
updated :)

agnieszkajacek commented:
Dołaczę do zapytania Tomka - brak zmian w `schema.rb`

Oraz zapytam jeszcze o to, gdzie się podział `Gemfile.lock` ?

Commit:
EK-427 Add comment_type, comment_user and Gemfile.lock

vitaliicherednichenko commented:
> Dołaczę do zapytania Tomka - brak zmian w `schema.rb`
> 
> Oraz zapytam jeszcze o to, gdzie się podział `Gemfile.lock` ?

`Gemfile.lock` znajdował się w .gitignore. 
Zaktualizowano

tamaraStrzeszewska23 commented on config/locales/pl.yml:
```diff
@@ -10,6 +10,7 @@ pl:
     add_comment: Dodaj komentarz
     add_comment_type: Dodaj typ komentarza
     comment_types: Rodzaje komentarzy
+    comment_user: Użytkownik komentarza
```
Brzmi trochę nie po polsku, nie chodzi przypadkiem o `Komentarz użytkownika`?

vitaliicherednichenko replied:
:) thanks!

tamaraStrzeszewska23 reviewed (commented):
Przydałby się też nowy filmik z testowania, bo zaszły znaczące zmiany imo

Commit:
EK-427 Add permitted_resource_params, change with_answer? and translation

Commit:
EK-427 Upgrade spree and ruby versions in REDME

Commit:
EK-427 Refactor migration

vitaliicherednichenko commented:
> Przydałby się też nowy filmik z testowania, bo zaszły znaczące zmiany imo

filmik nie różni się niczym poza innym miejscem formy i liczbą kolumn

<img width="1120" height="832" alt="Screenshot 2025-09-15 at 08 38 38" src="https://github.com/user-attachments/assets/e6318d71-c07f-439d-b197-5b8df8ac125e" />
